calibrate.py

```diff
@@ -5,27 +5,44 @@
 import fileinput
 import SPEFile
 import os
+import sys
+
 energy_list = [351.93, 583.19, 609.31, 911.20, 1460.82, 1764.49,
                2614.51]
 cal_headers = [351.93, 583.19, 609.31, 911.20, 1460.82, 1764.49,
                2614.51, 'Output']
 
-
 def acquire_files():
     """
     acquire_files gathers all the .Spe file in the current file directory and
     returns a list containing all .Spe files.
-    Files is a list of .spe files.
     """
     sample_measurements = []
+    sample_names = []
     dir_path = os.getcwd()
+    skip_file = ''
     for file in os.listdir(dir_path):
         if file.lower().endswith(".spe"):
-            if file == "USS_Independence_Background.Spe":
+            # Ignore the background and reference spectra
+            if file == skip_file:
+                pass
+            elif file == "USS_Independence_Background.Spe":
+                pass
+            elif file == "UCB018_Soil_Sample010_2.Spe":
                 pass
             else:
+                if '_recal' in file:
+                    skip_file = file
+                    skip_file.replace('_recal.Spe', '.Spe')
```

Comment:
This `skip_file` works if (for example) `UCB028_Black_Cod.Spe` comes later in the file list than `UCB028_Black_Cod_recal.Spe`. But if `_recal` comes later, then the non-`_recal` version is already in the file list.

You might put every `.spe` file in the file list first, and then remove the ones that are duplicated by `_recal` versions.